backend/typescript/rest/entityRoutes.ts

```diff
@@ -116,7 +116,7 @@ entityRouter.delete("/:id", async (req, res) => {
 
   try {
     await entityService.deleteEntity(id);
-    res.status(204).send();
+    res.status(200).send();
```

Comment:
We can return the deleted id now right?

```suggestion
    const deletedId = await entityService.deleteEntity(id);
    res.status(200).json({ id: deletedId });
```